Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptTypesManager.java

```diff
@@ -15,6 +17,9 @@
 	@Autowired
 	private TypeDatabaseClient client;
 
+    @Value("${default_page_size}")
+    private Integer defaultPageSize;
+
```

Comment:
I think we can format all the files once. The current setting has "spaces" formatting but existing file is formatted using tab spaces.